src/main/java/homework3/Task1.java

```diff
@@ -0,0 +1,18 @@
+package homework3;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int a = 13;
+        float b = 6.55f;
```

Comment:
"Create a program which has two integer numeric variables "
```suggestion
        int b = 6;
```